packages/application-components/src/components/internals/page-header.tsx

```diff
@@ -17,12 +18,18 @@ type Props = {
 
 const PageHeader = (props: Props) => {
   return (
+    // Bear in mind the paddings are dependant on the context the component
+    // is rendered in. For example, when rendered in a custom view, its panel
+    // already contains default paddings, so we don't need to add them here.
     <div
       css={css`
         margin: ${appKitDesignTokens.marginForModalPageHeader};
         padding: ${appKitDesignTokens.paddingForModalPageHeader};
         border-bottom: 1px solid
           ${appKitDesignTokens.borderColorForModalPageHeaderDivider};
+        body[data-extension-type='${CUSTOM_EXTENSION_TYPES.CUSTOM_VIEW}'] & {
```

Comment:
This is a bit "surprising" to me that we can target global elements within a scoped style.

I would suggest to follow this approach (@ByronDWall actually proposed it 😅): https://github.com/emotion-js/emotion/issues/2836#issuecomment-1406917978

```tsx
css`
  * :where([data-extension-type="custom-view"]) & {
    margin: ${appKitDesignTokens.paddingForModalPageHeaderInCustomView};
  }
`
```

Reply:
Updated here: cf1c6e521349c4a07621359d7b5c186cf97e4506